includes/my-orders-functions.php

```diff
@@ -0,0 +1,100 @@
+<?php
```

Comment:
Don't introduce unnecessary files. Instead of creating a new file for just one function, declare the function inside an existing relevant file.